src/leases/components/leaseSections/leaseArea/CustomDetailedPlan.js

```diff
@@ -0,0 +1,247 @@
+              {LeaseAreaCustomDetailedPlanFieldTitles.DETAILED_PLAN}
+            </FormTextTitle>
+            <FormText>{customDetailedPlan.detailed_plan || '-'}</FormText>
+          </Authorization>
+        </Column>
+        <Column small={12} medium={4} large={4}>
+          <Authorization allow={isFieldAllowedToRead(attributes, LeaseAreaCustomDetailedPlanFieldPaths.DETAILED_PLAN_LATEST_PROCESSING_DATE)}>
+            <FormTextTitle uiDataKey={getUiDataLeaseKey(LeaseAreaCustomDetailedPlanFieldPaths.DETAILED_PLAN_LATEST_PROCESSING_DATE)}>
+              {LeaseAreaCustomDetailedPlanFieldTitles.DETAILED_PLAN_LATEST_PROCESSING_DATE}
+            </FormTextTitle>
+            <FormText>{formatDate(customDetailedPlan.detailed_plan_latest_processing_date) || '-'}</FormText>
+          </Authorization>
+        </Column>
+        <Column small={12} medium={4} large={4}>
+          <Authorization allow={isFieldAllowedToRead(attributes, LeaseAreaCustomDetailedPlanFieldPaths.DETAILED_PLAN_LATEST_PROCESSING_DATE_NOTE)}>
+            <FormTextTitle uiDataKey={getUiDataLeaseKey(LeaseAreaCustomDetailedPlanFieldPaths.DETAILED_PLAN_LATEST_PROCESSING_DATE_NOTE)}>
+              {LeaseAreaCustomDetailedPlanFieldTitles.DETAILED_PLAN_LATEST_PROCESSING_DATE_NOTE}
+            </FormTextTitle>
+            <FormText>{customDetailedPlan.detailed_plan_latest_processing_date_note || '-'}</FormText>
+          </Authorization>
+        </Column>
+      </Row>
+      <Row>
+        <Column small={12} medium={4} large={4}>
+          <Authorization allow={isFieldAllowedToRead(attributes, LeaseAreaCustomDetailedPlanFieldPaths.RENT_BUILD_PERMISSION)}>
+            <FormTextTitle uiDataKey={getUiDataLeaseKey(LeaseAreaCustomDetailedPlanFieldPaths.RENT_BUILD_PERMISSION)}>
+              {LeaseAreaCustomDetailedPlanFieldTitles.RENT_BUILD_PERMISSION}
+            </FormTextTitle>
+            <FormText>
+              {!isEmptyValue(customDetailedPlan.rent_build_permission) ? `${formatNumber(customDetailedPlan.rent_build_permission)} m²` : '-'}
+            </FormText>
+          </Authorization>
+        </Column>
+        <Column small={12} medium={4} large={4}>
+          <Authorization allow={isFieldAllowedToRead(attributes, LeaseAreaCustomDetailedPlanFieldPaths.PRECONSTRUCTION_ESTIMATED_CONSTRUCTION_READINESS_MOMENT)}>
+            <FormTextTitle uiDataKey={getUiDataLeaseKey(LeaseAreaCustomDetailedPlanFieldPaths.PRECONSTRUCTION_ESTIMATED_CONSTRUCTION_READINESS_MOMENT)}>
+              {LeaseAreaCustomDetailedPlanFieldTitles.PRECONSTRUCTION_ESTIMATED_CONSTRUCTION_READINESS_MOMENT}
+            </FormTextTitle>
+            <FormText>{customDetailedPlan.preconstruction_estimated_construction_readiness_moment || '-'}</FormText>
+          </Authorization>
+        </Column>
+      </Row>
+      {/* Usage distributions (Käyttöjakaumat) */}
+      {!customDetailedPlan.usage_distributions.length ||
```

Comment:
IDK why, but it rendered the 0 when it was `&&` :D

Reply:
Ah, right. It's kinda the same case as the index ones later on, in that what we actually care about is whether the length is > 0 and not whether it's truthy (if usage_distributions exists, it should always be an array that has a length, I believe). But if it's just used as a number as is and not as a part of an expression returning a boolean, then it would just render that number indeed.

Reply:
Oh. Yeah. Now I get it. length === 0 and it will work. Will fix..